etl.js

```diff
@@ -146,7 +146,34 @@ Etl.upsertChecklistItem = async function _upsertChecklistItem(item) {
     );
     projectMeta.amount = item._amount;
     store.set(`${ISSUE_TO_ITEM}:${item.id}:project`, projectMeta);
-    await sleep(SLEEP);
+  }
+
```

Comment:
Note: Code should read like paragraphs. When you finish a thought, but a newline.

This seems a bit dense.